### PR TITLE
feat: add sqs queue for csv products

### PR DESCRIPTION
**Pull request by Arestout**
+ Configure the SQS to trigger lambda catalogBatchProcess with 5 messages at once
+ SNS topic subscription

Link to frontend: https://d3qsyl0aw7gjne.cloudfront.net/admin/products

### CODE REVIEW

File: import-service/serverless.ts [hunk 1]
```diff
@@ -29,15 +29,16 @@ const serverlessConfiguration: Serverless = {
...
     },
     iamRoleStatements: [
       { Effect: 'Allow',
         Action: 's3:ListBucket',
         Resource: [`arn:aws:s3:::${BUCKET}`]
       },
       { Effect: 'Allow',
-        Action: ['s3:*'],
-        Resource: [`arn:aws:s3:::${BUCKET}/*`]
+        Action: 'sqs:*',
```
Comment (aosin) [inline import-service/serverless.ts:51]: Shouldn't the import-service still have _write_ access to the bucket, to be able to move the file to /parsed?

File: product-service/handlers/catalogBatchProcess/catalogBatchProcess.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      console.log("SNS: message was sent:", data);
+    }
+  )
+  .promise();
+  next();
+})
+
+catalogBatchProcess
+  .use(sqsPartialBatchFailure())
+  .use(httpSecurityHeaders())
```
Comment (aosin) [inline product-service/handlers/catalogBatchProcess/catalogBatchProcess.ts]: You probably don't need any HTTP middleware here.

File: import-service/serverless.ts (hunk 1 above)
Comment (Arestout) [reply import-service/serverless.ts:51]: it has write access as all my files end up in the parsed folder

File: product-service/handlers/catalogBatchProcess/catalogBatchProcess.ts (hunk 2 above)
Comment (Arestout) [reply product-service/handlers/catalogBatchProcess/catalogBatchProcess.ts]: yes, good catch.

**Pull request merged**